identity/src/main/java/com/android/identity/Utility.java

```diff
@@ -316,16 +319,13 @@ byte[] encodeStaticAuthData(
 
             HashMap<String, List<byte[]>> issuerSignedMapping = new HashMap<>();
 
-            CborBuilder vdBuilder = new CborBuilder();
-            MapBuilder<CborBuilder> vdMapBuilder = vdBuilder.addMap();
-
             Iterator<Integer> digestIt = digestIds.iterator();
```

Comment:
Would you mind also changing this to use `Long` instead of `Integer` for the digest-ids? That way we avoid a cast below - thanks!